Cargo.toml

```diff
@@ -50,15 +50,15 @@ toml = { version = "0.7.2" }
 # v0.0.1
 libcst = { git = "https://github.com/charliermarsh/LibCST", rev = "80e4c1399f95e5beb532fdd1e209ad2dbb470438" }
 # v0.0.3
-ruff_text_size = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "f60e204b73b95bdb6ce87ccd0de34081b4a17c11" }
+ruff_text_size = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "caf6ebcefa80ce835aa689f392000e5cfd18f93d" }
 # v0.0.3
-rustpython-ast = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "f60e204b73b95bdb6ce87ccd0de34081b4a17c11" , default-features = false, features = ["all-nodes-with-ranges", "num-bigint"]}
+rustpython-ast = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "caf6ebcefa80ce835aa689f392000e5cfd18f93d" , default-features = false, features = ["num-bigint"]}
 # v0.0.3
-rustpython-format = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "f60e204b73b95bdb6ce87ccd0de34081b4a17c11", default-features = false, features = ["num-bigint"] }
+rustpython-format = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "caf6ebcefa80ce835aa689f392000e5cfd18f93d", default-features = false, features = ["num-bigint"] }
 # v0.0.3
-rustpython-literal = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "f60e204b73b95bdb6ce87ccd0de34081b4a17c11", default-features = false }
+rustpython-literal = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "caf6ebcefa80ce835aa689f392000e5cfd18f93d", default-features = false }
 # v0.0.3
```

Comment:
Can you first merge these changes, re-tag, and then update this PR with the updating tag?

Reply:
Oh. Are we crating tags now for every change? Makes sense. I'll add some documentation to make this clear because I didn't create a tag last time (I thought we only do so when merging RustPython but that's obviously not sufficient)

Reply:
Yeah, sorry, every time we update in Ruff we need to tag -- otherwise, we run the risk of breaking old Ruff builds when we rebase.

Reply:
I'll go through the last upgrades and create the necessary tags. Does cargo support tags?

Reply:
It does. I'll change it to use a tag instead

Reply:
I think if you use a tag, then every time you run a Cargo command, it does a lookup to match the tag to a SHA. We ran into this once and reverted it -- it was extremely annoying.

Reply:
Created tags from 0.0.3 to 0.06

Reply:
Tyvm